### DISCUSSION TITLE
Latest Release

### DISCUSSION THREAD

**Discussion by arunjayaraj1988** [WinMerge/winmerge#576] [General]
HI Winmerge Team,
Could you please let me know, when the latest version will be released? Any date please?

**Comment by sdottaka** [comment]
Today

**Reply by mlipok** [reply]
awesome news

**Reply by arunjayaraj1988** [reply]
With the new release, could you please let me know the command to get the dll version number also along with the report?

Now i run this, but the DLL version is not getting displayed in the report. 
Start-Process -FilePath $winMergeExePath -ArgumentList "$fullUNCLeftPath $fullUNCRightPath  -minimize -noninteractive -noprefs -cfg Settings/DirViewExpandSubdirs=1 -cfg ReportFiles/ReportType=2 -cfg ReportFiles/IncludeFileCmpReport=1 -r -or $reportPath" -wait

**Reply by sdottaka** [reply]
Please specify `-cfg "DirView/ColumnOrders=0 1 2 3 4 -1 -1 5 -1 -1 -1 -1 -1 6 7 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1` as shown in https://github.com/WinMerge/winmerge/issues/418#issuecomment-683424684.

~~~
Start-Process -FilePath $winMergeExePath -ArgumentList "$fullUNCLeftPath $fullUNCRightPath -minimize -noninteractive -noprefs -cfg Settings/DirViewExpandSubdirs=1 -cfg ReportFiles/ReportType=2 -cfg ReportFiles/IncludeFileCmpReport=0 -r -or $reportPath -cfg ""DirView/ColumnOrders=0 1 2 3 4 -1 -1 5 -1 -1 -1 -1 -1 6 7 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1""" -wait
~~~
